ui/cypress/e2e/collectors.test.ts

```diff
@@ -253,4 +253,101 @@ describe('Collectors', () => {
       })
     })
   })
+
+  describe('configuring plugins', () => {
+    // fix for https://github.com/influxdata/influxdb/issues/15500
+    describe('configuring nginx', () => {
+      beforeEach(() => {
+        cy.contains('Create Configuration').click()
```

Comment:
why this and not just `cy.visit(whateverurl)`? do we want to test that you always get to the page the same way? or is this some weirdness with the tests? also, idk what i'm reading

Reply:
this all happens in the context of a popups / modals - you click 'Create Configuration', then you click NGINX (or redis), then Cotinue, etc. i'll add a comment explaining it.

Reply:
might be fine as is. i was just under the impression that all of our modals are currently behind a route and got suspicious about state management within the app.

Reply:
@drdelambre depending on the url you are trying to visit since it might be outside of our control. Generally, if we're trying to test out whether or not we are visiting that other page (i.e. within our site) it's just easy to test out that other page separately rather than include that within the scope of this test